src/test/scholarshipsTest.js

```diff
@@ -0,0 +1,54 @@
+    },
+    {
+        id: "3",
+        name: "scholarship 3",
+        amount: 500,
+        deadline: "September 25, 2020",
+        website: "website 3",
+        school: "school 3",
+        year: "highschool senior",
+        description: "Cillum duis deserunt consectetur minim nulla voluptate excepteur occaecat laborum fugiat voluptate aute reprehenderit. Sint labore amet ipsum labore proident incididunt. Velit incididunt sit excepteur veniam pariatur. Culpa elit cupidatat est nostrud duis excepteur dolor cupidatat proident occaecat quis do. Fugiat consectetur ipsum dolore consectetur ex ea excepteur ut. Ex duis reprehenderit tempor proident duis ut et.Veniam cillum labore reprehenderit ipsum fugiat sit non. Ad magna cupidatat incididunt Lorem mollit. Nisi velit minim sit proident quis duis ex commodo amet ipsum. Ea officia laborum minim et elit excepteur amet id cupidatat nostrud." 
+    },
+    {
+        id: "4",
+        name: "scholarship 4",
+        amount: 900,
+        deadline: "November 12, 2020",
+        website: "website 4",
+        school: "school 4",
+        year: "college freshman",
+        description: "Tempor qui qui labore velit do voluptate nostrud occaecat elit pariatur tempor ad aute do. Incididunt voluptate dolore incididunt consequat officia esse non quis adipisicing nisi id pariatur. Ipsum ipsum et elit officia sint nulla qui officia magna.Aliquip laborum culpa aliquip minim culpa. Est laborum anim magna deserunt irure mollit pariatur velit ullamco. Nulla excepteur et enim amet labore quis in eiusmod consectetur." 
+    },
+    {
+        id: "5",
+        name: "scholarship 5",
+        amount: 1000,
+        deadline: "October 1, 2020",
+        website: "website 5",
+        school: "school 5",
+        year: "college freshman",
+        description: "Sit ipsum magna elit deserunt dolor aliquip nostrud eu. Enim dolor anim voluptate consectetur aute. Id minim cillum excepteur enim quis enim enim quis. Nulla veniam ullamco dolore dolore nisi.Ea eiusmod laborum sit sit officia tempor fugiat labore cupidatat nisi eu consequat culpa. Dolore consectetur irure do commodo sunt aliquip occaecat esse reprehenderit ipsum sit Lorem consequat. Ipsum sunt nulla incididunt nulla elit labore anim in dolore quis anim proident. Laborum dolor ad commodo amet ad nostrud anim consectetur adipisicing minim voluptate velit Lorem magna. Sunt enim dolor do deserunt veniam ipsum dolor occaecat voluptate tempor aliquip excepteur nulla. Ea laboris minim Lorem consequat sunt anim anim laboris aliquip mollit anim. Duis ut cillum nisi mollit eiusmod." 
+    }
+];
```

Comment:
(Optional) Since these scholarships are all basically the same, an alternative might be to write a `fakeScholarship()` and/or `fakeScholarshipWithId(id)` functions that return a random scholarship. Check out [this gist](https://gist.github.com/miguelmota/5b67e03845d840c949c4) to see how to randomize dates, and [this page](https://www.w3schools.com/js/js_random.asp) for randomizing numbers